stubs/psutil/psutil/_common.pyi

```diff
@@ -193,8 +195,8 @@ class pctxsw(NamedTuple):
 
 class pconn(NamedTuple):
     fd: int
-    family: Any
-    type: Any
+    family: AddressFamily
+    type: SocketKind
```

Comment:
Which function returns `pconn` instances? The stubs claim that there is `psutil.connections()`, but it doesn't actually exist at runtime.

Reply:
Nevermind. I realized that `connections()` is a method of `Process` and I checked that the annotations are correct:

```
>>> psutil.Process(os.getpid()).connections()
[]
>>> import socket
>>> s=socket.socket()
>>> s.connect(('www.google.com', 80))
>>> psutil.Process(os.getpid()).connections()
[pconn(fd=3, family=socket.AF_INET, type=socket.SOCK_STREAM, laddr=addr(ip='192.168.1.3', port=45972), raddr=addr(ip='142.250.74.100', port=80), status='ESTABLISHED')]
```